samples/cpp/benchmark_app/main.cpp

```diff
@@ -161,12 +190,16 @@ int main(int argc, char* argv[]) {
 
         // ----------------- 1. Parsing and validating input arguments
         // -------------------------------------------------
+        auto args_string = get_console_command(argc, argv);
```

Comment:
It should be called right before being used.
If it need to be called before parsing and checking put a comment why it is so.

Reply:
Parsing seems to modify the argv array. If get_console_command is executed before the gflags::ParseCommandLineNonHelpFlags(), the original command is printed correctly. If executed after, then all flag values seem to change to the argv[0] value